tests/sync/assessmentsSync.js

```diff
@@ -485,10 +485,38 @@ describe('Assessment syncing', () => {
     courseData.courseInstances[util.COURSE_INSTANCE_ID].assessments['fail'] = assessment;
     await util.writeAndSyncCourseData(courseData);
     const syncedAssessment = await findSyncedAssessment('fail');
-    assert.match(syncedAssessment.sync_errors, /Cannot specify an array of points for a question/);
+    assert.match(syncedAssessment.sync_errors, /Cannot specify an array of multiple point values for a question/);
   });
 
-  it('records an error if points array is specified for an alternative when real-time grading is disallowed', async () => {
+  it('accepts a points array of size 1 being specified for a question when real-time grading is disallowed', async () => {
```

Comment:
```suggestion
  it('accepts a single-element points array being specified for a question when real-time grading is disallowed', async () => {
```
nit: mirroring the multiple-element wording